orderer/common/multichannel/chainsupport.go

```diff
@@ -180,3 +182,18 @@ func (cs *ChainSupport) Sequence() uint64 {
 func (cs *ChainSupport) Append(block *cb.Block) error {
 	return cs.ledgerResources.ReadWriter.Append(block)
 }
+
+func newOnBoardingChainSupport(
+	ledgerResources *ledgerResources,
+	config localconfig.TopLevel,
+	bccsp bccsp.BCCSP,
+) (*ChainSupport, error) {
+	cs := &ChainSupport{ledgerResources: ledgerResources}
+	cs.Processor = msgprocessor.NewStandardChannel(cs, msgprocessor.CreateStandardChannelFilters(cs, config), bccsp)
```

Comment:
I would have thought this field would be safe to leave uninitialized? What path is the processor for this channel called?  If it's in the broadcast/deliver path, perhaps we'd be better off explicitly returning an error of some sort?

Reply:
It is in the broadcast path. The first is "ClasifyHeader" and then "Process*" which will ultimately get to the inactive.Chain which is primed with an informative error message. We'll test that in IT.